From af33b09d1f21eb3698c4977a4345b6badbefccc2 Mon Sep 17 00:00:00 2001
From: Nuno Araujo <nuno.araujo@russo79.com>
Date: Wed, 23 Nov 2011 18:30:28 +0100
Subject: [PATCH] Fix user-theme extension UUID

Since version 3.2.1, gnome-shell extensions changed their UUID to another value.
According to the changelog, this was made because gnome.org is temporarily
forbidden in UUIDS.

As a consequence, gnome-tweak-tool didn't detect the user-theme extension.

https://bugzilla.gnome.org/show_bug.cgi?id=664655
---
 gtweak/tweaks/tweak_shell.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/gtweak/tweaks/tweak_shell.py b/gtweak/tweaks/tweak_shell.py
index 4de335a..75dc5d8 100644
--- a/gtweak/tweaks/tweak_shell.py
+++ b/gtweak/tweaks/tweak_shell.py
@@ -43,7 +43,7 @@ class ShowWindowButtons(GConfComboTweak):
 
 class ShellThemeTweak(Tweak):
 
-    THEME_EXT_NAME = "user-theme@gnome-shell-extensions.gnome.org"
+    THEME_EXT_NAME = "user-theme@gnome-shell-extensions.gcampax.github.com"
     THEME_GSETTINGS_SCHEMA = "org.gnome.shell.extensions.user-theme"
     THEME_GSETTINGS_NAME = "name"
     THEME_DIR = os.path.join(GLib.get_home_dir(), ".themes")
-- 
1.7.8.1

